ecosystem/sep-0036.md

```diff
@@ -0,0 +1,137 @@
+## Preamble
+
+```
+SEP: 0036
+Title: Regulated Assets Transacting
+Author: Howard Chen <@howardtw>
+Status: Draft
+Created: 2020-10-26
+Version 0.1.0
+```
+
+## Simple Summary
+
+Regulated Assets provide ecosystem support for assets that require an issuer’s approval (or a delegated third party’s approval, such as a licensed securities exchange) on a per-transaction basis. It standardizes the identification of such assets as well as defines the protocol for performing compliance checks and requesting issuer approval.
+
+This SEP is based on [SEP-0008](sep-0008.md), but the issuer no long has to a co-signer on the account holding the regulated asset. It also updates the API of the approval server to better align with other active SEPs.
+
+**Target Audience**: Asset issuers, issuance platforms, wallet developers, and exchanges
+
+## Motivation
+
+Stellar aims to be an ideal platform for issuing securities. As such, it must provide tools to comply with various regulatory requirements. Regulation often requires asset issuers to monitor and approve every transaction involving the issued asset and to enforce an assortment of constraints. This SEP provides support for these requirements.
+
+## Overview
+
+Implementing a Regulated Asset consists of these parts:
+- [Stellar.toml]: Issuers will advertise the existence of an Approval Service and the approval criteria via their stellar.toml file.
+- [Approval Server]: HTTP protocol for transaction signing.
+- [Transaction Composition]: Wallets will have to put operations that fully authorize the accounts for the asset, transact the asset, and deauthorize the accounts in a single transaction.
+
+## Regulated Assets Approval Flow
+
+1. User creates and signs a transaction.
+2. Wallet resolves asset information and detects that it's a regulated asset.
+3. Wallet sends the transaction to the approval server.
+4. Approval server determines whether the transaction is compliant based on the compliance ruleset and looks up the current state of the ledger if necessary.
+5. Wallet handles approval response:
+    1. *Success?* Transaction has been approved and signed by issuer.
+    2. *Rejected?* Wallet should display given error message to the user.
+
+## Stellar.toml
```

Comment:
This seems to be redundant, the same definition is used earlier

Reply:
This section not only contains a short description but subsections like `Fields` and `Example`. The description is a little more descriptive than what's in the overview.